### ISSUE TITLE
2.6.0 version latest in PyPi

### ISSUE THREAD

**Issue by SShatun** [brunns/mbtest#43]
As I can see the latest version on that moment is 2.5.2

But the latest version in PyPi is 2.6.0 -  https://pypi.org/project/mbtest/

And if you run `pip install mbtest` it will install 2.6.0 version:
```
Installing collected packages: mbtest
  Attempting uninstall: mbtest
    Found existing installation: mbtest 2.5.2
    Uninstalling mbtest-2.5.2:
      Successfully uninstalled mbtest-2.5.2
Successfully installed mbtest-2.6.0
```

**Comment by brunns** [comment]
Argh, don't know what happened there...

**Comment by brunns** [comment]
Released 2.6.1.
